managed/services/management/management.go

```diff
@@ -25,8 +25,15 @@ import (
 	"github.com/percona/pmm/api/inventorypb"
 	"github.com/percona/pmm/api/managementpb"
 	"github.com/percona/pmm/managed/models"
+	managementbackup "github.com/percona/pmm/managed/services/management/backup"
 )
 
+type MgmtServices struct {
+	BackupsService        *managementbackup.BackupsService
+	ArtifactsService      *managementbackup.ArtifactsService
+	RestoreHistoryService *managementbackup.RestoreHistoryService
```

Comment:
Maybe it makes sense to merge these 3 services into single one and do the same with API. Not sure, but it seems that we have too much in common between them.

Reply:
Agree, on API level we can merge the functionality.